doc/source/notes/5.0.0.rst

```diff
@@ -15,6 +15,7 @@ Features
 
 * `OpenStack Services Action <https://github.com/marketplace/actions/openstack-services-action>`_ now
   uses OpenStack Zed as the default OpenStack release
+* OpenStack client container image in version Antelope is available
```

Comment:
Antelope is to be released in 03/2023, so I very much doubt this. Or do you plan to release 5.0.0 only after the OpenStack 2023.1 release?

Reply:
Hm is goofy from the wording. I had also done that with Zed so that I added Zed and used Master for it until the release. Thought that we do the same for Antelope and then change from master to stable/antelope after the release.